### PR TITLE
Feature/1208 enhance get settlement by id to include window content

### PR DESCRIPTION
**Pull request by bothadeon**
Enhance central-settlement : Return settlement window content for every settlement window when getting settlements  by id.
 
1. Updated dependancies in package.json.
2. Updated the version to 9.1.0-snapshot.
3. Add code to get src/domain/settlement/index.js  : getById to add the window content to the current response object.
4. Update unit testing and code coverage for added code.

### CODE REVIEW

File: src/domain/settlement/index.js [hunk 1]
```diff
@@ -89,6 +95,7 @@ module.exports = {
     const settlement = await SettlementModel.getById({ settlementId }, enums)
     if (settlement) {
       const settlementWindowsList = await SettlementWindowModel.getBySettlementId({ settlementId }, enums)
+      await prepareSettlementWindowsListWithContents(settlementWindowsList)
```
Comment (ggrg) [inline src/domain/settlement/index.js]: It is generally a bad idea to implement functions with side effects. Please, change to pure function, or iterate windows list here, or modify SettlementWindowModel.getBySettlementId if used only here.

**Comment by ggrg** [comment]
Also, please add PR description, connect to story and bring your branch up-to-date with settlement-v2.

File: src/domain/settlement/index.js (hunk 1 above)
Comment (bothadeon) [reply src/domain/settlement/index.js]: Implemented a change as suggested

**Comment by bothadeon** [comment]
Add description.

**Comment by ggrg** [comment]
@bothadeon, I'm afraid I can not review before we upgrade all services (central-ledger, simulator, central-ledger) to Node v12. Please, put the story to blocked and dependent on story #1175. When @oderayi is ready, I'll merge the changes from master to settlement-v2 branches, then you'll need to update this branch and make sure it's running properly and only then I'll be able to verify the PR. Thanks.

**Comment by bothadeon** [comment]
Create a new pull request after local branch got corrupted. refer to : https://github.com/mojaloop/central-settlement/pull/255

**Comment by rmothilal** [comment]
@bothadeon is this still valid?

**Comment by rmothilal** [comment]
closing replaced by pull request #255

**Pull request closed without merging**